components/automate-ui/src/app/pages/top-nav-landing/top-nav-landing.component.ts

```diff
@@ -0,0 +1,65 @@
+import { Component } from '@angular/core';
+import { RoutePerms } from 'app/components/landing/landing.component';
+import { LayoutFacadeService } from 'app/entities/layout/layout.facade';
+
+@Component({
+  selector: 'app-top-nav-landing',
+  templateUrl: './top-nav-landing.component.html',
+  styleUrls: []
+})
+export class TopNavLandingComponent {
+
+  // order must match layout in navbar.component.html
```

Comment:
is there a possible refactor so that we only save these checks in one place?

Reply:
Great question! For the sidebars, yes, I am going to do that. For the top nav bar, we will be able to do that when the top bar gets migrated into the sidebar.